sysdig/resource_sysdig_team_service_account_test.go

```diff
@@ -0,0 +1,97 @@
+//go:build tf_acc_sysdig_monitor || tf_acc_sysdig_secure
```

Comment:
Is the feature available for IBM?

Reply:
For now we don't support team service account in IBM envs.